### DISCUSSION TITLE
Auffrischungsimpfung

### DISCUSSION THREAD

**Discussion by TH-Pega** [Digitaler-Impfnachweis/certification-apis#178] [Ideas]
Im September sollen die Auffrischungsimpfungen starten. Gibt es schon Vorstellungen, wie diese als Zertifikat abzubilden sind?

**Comment by mge-medisoftware** [comment]
Seit 1.9. ist Auffrischimpfung möglich, Praxen fragen schon nach..
Für Auffrischimpfungen soll nach Dokument im Anhang nachfolgend verfahren werden:

(VERALTET! Siehe weiter unten...)

Booster vaccination will increase numerator. Denominator stays unchanged. 2/2 -> 3/2

Booster of Astrazeneca, Pfizer, Moderna  3/2
Booster of Cross Vaccination eg. Astrazeneca -> Pfizer  3/2
Second Booster of Astrazeneca, Pfizer, Moderna   4/2
Booster of J&J  2/1
Booster after Recovery Vaccination  2/1

[BoosterVaccination.pdf](https://github.com/Digitaler-Impfnachweis/certification-apis/files/7098233/BoosterVaccination.pdf)

Beim Senden der Impfdaten auf "Produktionsumgebung" gibt es jedoch einen Fehler:

HTTP 400
Sorry, there is something invalid in your request: Current dose number (3) can't exceed the total number of doses (2)


Wann kommt eine angepasste Version?

Referenz konnte ich nicht testen, da scheinbar neuerdings (ca. 1 Woche) Zertifikat von unserer gematik-Test-SMC-B nicht mehr akzeptiert wird.

**Reply by Ein-Tim** [reply]
Scheinbar ist es einigen Stellen schon Möglich "2/3" Zertifikate auszustellen: Ein Twitter Nutzer hat [hier](https://twitter.com/hfeddersen/status/1431207938924990472?s=21) dieses Bild mit mir geteilt: 
![image](https://user-images.githubusercontent.com/67682506/131831662-c03dfe20-c715-4e86-85ec-fd1adb938fc2.jpeg)

**Comment by MikeMcC399** [comment]
@Ein-Tim 
A 2/3 certificate is bad for the holder, because it means that the series is incomplete. At least they should have a 2/2 certificate to fall back on! Do you know where this came from? Impfzentrum, Apotheke or Arztpraxis?

**Reply by Ein-Tim** [reply]
@MikeMcC399, yes, I know, that's why I opened: https://github.com/corona-warn-app/cwa-app-ios/issues/3436
I think version 2.10 of the Corona-Warn-App will be able to detect "2/3" certificates and show them ad valid. The user [writes](https://twitter.com/hfeddersen/status/1431207100219265027?s=21) that he has received the certificate in the vaccination center Norderbrarup.

**Comment by mge-medisoftware** [comment]
Hallo, wie Murphy es so treibt....

Habe gerade ein neues Dokument erhalten, demnach muss der "Nenner" auch inkrementiert werden!

[BMG -Vertrag Modul Covid-19 Zertifikate - Anlage 1.2.1 - Leistungsbeschreibung_Auffrischungsimpfung_20210827_Fin.pdf](https://github.com/Digitaler-Impfnachweis/certification-apis/files/7098844/BMG.-Vertrag.Modul.Covid-19.Zertifikate.-.Anlage.1.2.1.-.Leistungsbeschreibung_Auffrischungsimpfung_20210827_Fin.pdf)

Also 1. Auffrischung nach vollständiger Impfung mit BioNTech, Astrazeneca oder Moderna wäre dann 3/3 !

**Comment by Ein-Tim** [comment]
Any update on this?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Did you see the sentence in the published [Corona-Warn-App 2.9 Privacy Notice - EN](https://www.coronawarn.app/assets/documents/cwa-privacy-notice-en.pdf)?

"There is currently no EU-wide agreement on booster vaccinations, so booster vaccination certificates may not be recognised as valid in other EU countries."

or [Corona-Warn-App 2.9 Privacy Notice - DE](https://www.coronawarn.app/assets/documents/cwa-privacy-notice-de.pdf)

"Zu Auffrischungsimpfungen gibt es derzeit noch keine EU-weite Einigung, weshalb Zertifikate zu Auffrischungsimpfungen in anderen EU-Ländern möglicherweise nicht als gültig anerkannt werden."

**Reply by Ein-Tim** [reply]
@MikeMcC399

No, I did not see this. Thank you! 

However, it would still be good to know whether it's planned and possible for German authorities to issue "3/3" certificates.

**Comment by Ein-Tim** [comment]
FYI: https://www.apotheke-adhoc.de/nachrichten/detail/apothekenpraxis/startschuss-fuer-die-booster-zertifikate-update-des-dav-portals/?utm_source=social&utm_medium=social&utm_campaign=15092021 sagt: 

> Über das DAV-Portal können nun auch die digitalen Zertifikate für Booster-Impfungen ausgestellt werden. [...]
Dies geschieht wie gewohnt über das Feld zur Eingabe der Dosis-Nummer: Hier kann nun die Booster-Dosis ausgewählt werden. Je nach ausgewählter Impfung und den benötigten Impfdosen wird das Feld unterschiedlich angezeigt: Bei Vaxzevria, Comirnaty und Spikevax gilt die Auffrischimpfung als „3/3“, bei einer Impfung mit dem Vakzin von Janssen entsprechend als „2/2“.

Beantwortet das die hier gestellte Frage @TH-Pega?

**Comment by mwicke** [comment]
Wir haben hier eine Beschreibung (die wir aktuell halten), wie u.A. Booster-Impfungen in die Zertifikate einzutragen sind: https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/Implementation.md
